### ISSUE TITLE
Useless loadLanguageFile call when using defaults

### ISSUE THREAD

**Issue by Toflar** [contao/core#7805]
When I use `{{label::MSC:search}}` it works (because `default.php` is always loaded) but there's a useless `System::loadLanguageFile()` call because it tries to load `System::loadLanguageFile('MSC')`. See https://github.com/contao/core/blob/release/3.5.0/system/modules/core/library/Contao/InsertTags.php#L196

**Comment by leofeyer** [comment]
Do you have a fix in mind?

**Comment by Toflar** [comment]
Well, the way it was implemented in the first place doesn't allow anything else than excluding the known keys like `MSC`.

**Comment by leofeyer** [comment]
So how would you change it? :)

**Comment by Toflar** [comment]
```php
case 'MSC':
case 'ERR':
case 'CTE':
case 'PTY':
case 'FOP':
case 'CHMOD':
case 'DAYS':
case 'MONTHS':
case 'UNITS':
case 'CONFIRM':
case 'DP':
case 'COLS':
    $file = 'default';
    break;
```

and

```php
if ('default' !== $file) {
    \System::loadLanguageFile($file);
}

**Comment by leofeyer** [comment]
Fixed in 75c6f5f90021d5bb77945fd669134613546b7ab0. I did not add the if condition, because `System::loadLanguageFile()` will know that the file has been loaded before (see https://github.com/contao/core/blob/release/3.5.0/system/modules/core/library/Contao/System.php#L276).

**Comment by Toflar** [comment]
Thx!
